ibis/backends/snowflake/__init__.py

```diff
@@ -228,6 +231,53 @@ def normalize_name(name):
         self.con.dialect.normalize_name = normalize_name
         return res
 
+    def _compile_scalar_udf(self, udf_node: ops.ScalarUDFNode):
+        dialect = self.con.dialect
+        name = type(udf_node).__name__
+        signature = ", ".join(
+            f"{name} {sa.types.to_instance(to_sqla_type(dialect, arg.output_dtype)).compile(dialect=dialect)}"
+            for name, arg in zip(udf_node.argnames, udf_node.args)
+        )
+        snowflake_type = sa.types.to_instance(
+            to_sqla_type(dialect, udf_node.output_dtype)
+        ).compile(dialect=dialect)
+        source = textwrap.dedent(inspect.getsource(udf_node.__func__)).strip()
+        source = "\n".join(
+            line for line in source.splitlines() if not line.startswith("@")
```

Comment:
Decorators show up in the source, but we don't want the `@udf` one to show up because it's coming from ibis.

We should probably _just_ ignore that one and not all decorators.

Reply:
Done.